src/components/ui4/CardUi4.tsx

```diff
@@ -78,7 +78,7 @@ export const CardUi4 = (props: Props) => {
       try {
         await onLongPress()
       } catch (err) {
-        showError(err)
+        showDevError(err)
```

Comment:
Since this is already inside `EdgeTouchableOpacity`, just delete the whole try / catch block.